go/pkg/chunker/chunker.go

```diff
@@ -77,7 +76,7 @@ func New(ue *uploadinfo.Entry, compressed bool, chunkSize int) (*Chunker, error)
 			chunkSize = IOBufferSize
 		}
 	} else {
-		return nil, errors.New("invalid Entry")
+		return nil, errors.New(fmt.Sprintf("invalid entry: BlobLength=%d, PathLength=%d", ue.BlobLen(), len(ue.Path())))
```

Comment:
What's the value of PathLength?

Reply:
This was based on a past iteration where I've attempted to get rid of the internal type field. I should change this line to print the entry type here.